lib/matplotlib/table.py

```diff
@@ -25,6 +25,7 @@
 """
 
 import numpy as np
+import pandas as pd
```

Comment:
pandas is an optional dependency for matplotlib. We cannot rely that pandas is available. (It is in the test environment so that we can test against it).

Reply:
Gotcha. Ill remove it and find another way